src/Microsoft.PowerShell.Commands.Management/commands/management/PingPathCommand.cs

```diff
@@ -154,12 +162,39 @@ internal override object GetDynamicParameters(CmdletProviderContext context)
         /// </summary>
         protected override void ProcessRecord()
         {
+            if (_paths == null || _paths.Length == 0)
+            {
+                WriteError(new ErrorRecord(
+                    new ArgumentNullException("The path was null or an empty collection."),
```

Comment:
English text needs to be from a .resx file

Reply:
Done!